ruby/lib/ci/queue/configuration.rb

```diff
@@ -14,7 +14,7 @@ def from_env(env)
       end
 
       def initialize(
-        timeout: 10, build_id: nil, worker_id: nil, max_requeues: 0, requeue_tolerance: 0,
+        timeout: 30, build_id: nil, worker_id: nil, max_requeues: 0, requeue_tolerance: 0,
```

Comment:
Why this change?

Reply:
Because I figured 10 seconds is very low, and people trying out this gem might be surprised by the behavior. 30 seemed fair.